src/electron/views/screenshot/screenshot-container.tsx

```diff
@@ -2,16 +2,16 @@
 // Licensed under the MIT License.
 import { NamedFC } from 'common/react/named-fc';
 import { ScreenshotData } from 'common/types/store-data/unified-data-interface';
-import { BoundingRectangle } from 'electron/platform/android/scan-results';
 import { HighlightBox } from 'electron/views/screenshot/highlight-box';
 import { Screenshot } from 'electron/views/screenshot/screenshot';
 import { screenshotContainer } from 'electron/views/screenshot/screenshot-container.scss';
+import { HighlightBoxViewModel } from 'electron/views/screenshot/screenshot-view-model';
 import { isEmpty } from 'lodash';
 import * as React from 'react';
 
 export interface ScreenshotContainerProps {
     screenshotData?: ScreenshotData;
-    highlightBoxRectangles: BoundingRectangle[];
+    highlightBoxViewModels: HighlightBoxViewModel[];
```

Comment:
does it make sense to make this new property optional the same as `screenshotData`? Alternative, does it make sense to make `screenshotData` mandatory to match the new one? I feel this 2 props are related in a way that you either have both or none, is this assumtion right?

Reply:
I like making it mandatory. Moved the responsibility for laying out the unavailable message to the view to accommodate this.